tests/Database/Behaviors/EncryptableBehaviorTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+use Illuminate\Encryption\Encrypter;
+
+class EncryptableBehaviorTest extends DbTestCase
+{
+    const TEST_CRYPT_KEY = 'gBmM1S5bxZ5ePRj5';
+
+    /**
+     * @var \Illuminate\Encryption\Encrypter Encrypter instance.
+     */
+    protected $encrypter;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+        $this->createTable();
+
+        $this->encrypter = new Encrypter(self::TEST_CRYPT_KEY, 'AES-128-CBC');
+    }
+
+    public function testEncryptableBehavior()
+    {
+        $testModel = new TestModelEncryptableBehavior();
+        $testModel->setEncrypter($this->encrypter);
+
+        $testModel->fill(['secret' => 'test']);
+        $this->assertEquals('test', $testModel->secret);
+        $this->assertNotEquals('test', $testModel->attributes['secret']);
+        $payloadOne = json_decode(base64_decode($testModel->attributes['secret']), true);
+        $this->assertEquals(['iv', 'value', 'mac', 'tag'], array_keys($payloadOne));
+
+        $testModel->secret = '';
+        $this->assertEquals('', $testModel->secret);
+        $this->assertNotEquals('', $testModel->attributes['secret']);
+        $payloadTwo = json_decode(base64_decode($testModel->attributes['secret']), true);
+        $this->assertEquals(['iv', 'value', 'mac', 'tag'], array_keys($payloadTwo));
+        $this->assertNotEquals($payloadOne['value'], $payloadTwo['value']);
+
+        $testModel->secret = 0;
+        $this->assertEquals(0, $testModel->secret);
+        $this->assertNotEquals(0, $testModel->attributes['secret']);
+        $payloadThree = json_decode(base64_decode($testModel->attributes['secret']), true);
+        $this->assertEquals(['iv', 'value', 'mac', 'tag'], array_keys($payloadThree));
+        $this->assertNotEquals($payloadTwo['value'], $payloadThree['value']);
+
+        $testModel->secret = null;
+        $this->assertNull($testModel->secret);
+        $this->assertNull($testModel->attributes['secret']);
+    }
+
+    protected function createTable()
+    {
+        $this->getBuilder()->create('secrets', function ($table) {
+            $table->increments('id');
+            $table->string('secret');
+            $table->timestamps();
+        });
+    }
+}
+
+class TestModelEncryptableBehavior extends \Winter\Storm\Database\Model
+{
+    public $implement = [
+        \Winter\Storm\Database\Behaviors\Encryptable::class,
+    ];
+
+    public $encryptable = ['secret'];
```

Comment:
```suggestion
    protected $encryptable = ['secret'];
```

Reply:
@mjauvin If the above closures you suggested will work, this can be done.

Reply:
It should work with latest develop branch for sure.